core/src/main/java/org/apache/accumulo/core/util/Timer.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.util;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class provides a timer for measuring elapsed time.
+ */
+public final class Timer {
+
+  private long startNanos;
+
+  private Timer() {
+    this.startNanos = System.nanoTime();
+  }
+
+  /**
+   * Creates and starts a new Timer instance.
+   *
+   * @return a new Timer instance that is already started.
+   */
+  public static Timer startNew() {
```

Comment:
Could rename this to `start()` since the other `start()` method was deleted.

Reply:
I kind of like `startNew()` as it implies a new object is being created and started